.npmignore

```diff
@@ -0,0 +1,11 @@
+# Visual Studio Code temporary files
+.vscode/
+
+# npm packages
+node_modules
+
+# npm logs
+npm-debug.log*
+
+# rollup-plugins-typescript2
+.rpt2_cache
```

Comment:
Generally yes - will need to check if there's a better way to keep these in sync.